docker/docker-compose-multiclusters.yml

```diff
@@ -44,21 +44,21 @@ services:
   cadence-secondary:
     image: ubercadence/server:master-auto-setup
     ports:
-      - "9000:9000"
       - "9001:9001"
       - "9002:9002"
       - "9003:9003"
+      - "9004:9004"
       - "7943:7933"
       - "7944:7934"
       - "7945:7935"
       - "7949:7939"
       - "7843:7833"
     environment:
       - "CASSANDRA_SEEDS=cassandra"
-      - "PROMETHEUS_ENDPOINT_0=0.0.0.0:9000"
```

Comment:
avoiding usage of port 9000 because it's a popular port and high chance of conflict. (I encountered this issue)